src/components/PostInfo/PostInfo.jsx

```diff
@@ -1,3 +1,29 @@
-export const PostInfo = () => (
-  <>Put the post here</>
-);
+import { CommentList } from '../CommentList';
+import { UserInfo } from '../UserInfo';
+import './PostInfo.scss';
+
+export const PostInfo = ({ post }) => {
+  const { title, body, comments, user } = post;
+  const commentsBlock = comments[0]
+    ? <CommentList comments={comments} />
+    : (
+      <>
+        <hr />
+        <b data-cy="NoCommentsMessage">No comments yet</b>
+      </>
+    );
```

Comment:
It is better to put it in JSX instead of separate variable
Because Now I need to scroll to understand the final markup

Also comment[0] is not really clear condition
It is better to use comments.length > 0